helm/hookshot/templates/_pod.tpl

```diff
@@ -48,6 +48,9 @@ containers:
         mountPath: "/data"
 {{- end }}
     ports:
+      - name: bridge
+        containerPort: {{ .Values.hookshot.config.bridge.port }}
```

Comment:
Heya, do you know what the bridge port corresponds to. This isn't the appservice (below)?

Reply:
Originally I thouth so, but I'm not sure, TBH. There is a port 9993 defined in the bridge configuration section. But in the service and container definitions are used hardcoded ports which does not correspond with bridge port. 🤷‍♂️

Reply:
So I've been digging more and the ports are really different. The `appservice` here is probably serving widget stuff, but the bridge itself is running on port defined in `.Values.hookshot.config.bridge.port`. Therefore, there must be another service for it.
Here is what ports are actually listening in the container:
![image](https://github.com/matrix-org/matrix-hookshot/assets/445003/955c3e77-c897-42ac-805b-d5cc5899cc45)

And [Here](https://github.com/matrix-org/matrix-hookshot/blob/958743365f02b33dc6293ea226c1d6b8b14b0245/helm/hookshot/values.yaml#L64) in the `values.yaml`, there is `appservice` as listener.
So I think this PR is legit. 🤔 At least I couldn't make the hookshot work without these changes.

Reply:
Hey, It's been a while. Did you manage to get into this? I want to use the Helm chart installation, but I cannot, since the bridge port is not propagated on `Pod` or even available in the `Service`.

If I am doing something wrong, I would like to know what 🤷‍♂️ 
Thanks

Reply:
Yes, can that please merged? We need to have Port 9993 opened in the cluster, so that we can reach it with the registration of synapse. Otherweise the helm chart ist useless.